src/mnesia_rocksdb.erl

```diff
@@ -182,6 +182,7 @@
 	    , type
 	    , size_warnings			% integer()
 	    , maintain_size			% boolean()
+            , on_write_error = verbose          :: debug | verbose | warning | error | fatal
```

Comment:
```suggestion
          , on_write_error = verbose          :: debug | verbose | warning | error | fatal
```

Reply:
The possible values should be documented. You already described some of them in the PR.